dwave_networkx/drawing/chimera_layout.py

```diff
@@ -141,10 +151,19 @@ def chimera_node_placer_2d(m, n, t, scale=1., center=None, dim=2):
     """
     import numpy as np
 
+    line_plot = False if normalize_kwargs is None else normalize_kwargs.get('line_plot')
```

Comment:
In all the user functions (e.g. `draw_pegasus`) the `line_plot` is described as a Boolean but actually just its existence is checked so `dnx.draw_chimera(G, line_plot=3.14)` would work. Not sure if you want to enforce or not.

Reply:
I do not want to enforce that.  If a user puts a number into a boolean field, they should expect that 0 will be treated as False and anything else should be treated as True.  In the case they have omitted `line_plot`, then the default value of `None` will be returned from `get('line_plot')`, which is in turn interpreted as `False`.